query/common.d.ts

```diff
@@ -4,10 +4,15 @@
 import { EventEmitter } from "events";
 import * as RDF from '../data-model';
 
+/**
+ * Helper union type for triple term names.
+ */
+export type TripleTermName = 'subject' | 'predicate' | 'object';
+
 /**
  * Helper union type for quad term names.
  */
-export type QuadTermName = 'subject' | 'predicate' | 'object' | 'graph';
+export type QuadTermName = TripleTermName | 'graph';
```

Comment:
I never realised we **did** have `QuadTermName` 😅

Reply:
Neither did I until I tried to add the same thing lol